packages/experience-builder-sdk/jest.config.js

```diff
@@ -10,4 +10,7 @@ export default {
     '^.+\\.svg$': '<rootDir>/test/fileMock.ts',
     '^.+\\.svg\\?react$': '<rootDir>/test/fileMock.ts',
   },
+  transform: {
+    '\\.[jt]sx?$': 'ts-jest',
```

Comment:
What problem is it solving?

Reply:
So as we are importing types from `experience-builder-types` repo. It exposes types in dist folder with js file, We need to tell jest to compile them as ES6, as they are not commonJs js files.

Reply:
Is there a way to build both - es6 and cjs files? Maybe vite provides such ability

Reply:
Not sure about vite, but swc builds for both and its lightweight than vite